.github/workflows/v8upgrade.yml

```diff
@@ -0,0 +1,52 @@
+name: V8 Upgrade
+
+on:
+  schedule:
+    - cron: '0 0 * * *' # Run every day
+
+jobs:
+    upgrade:
+        name: Upgrade V8
+        runs-on: ubuntu-18.04
+        steps:
+            - name: Checkout
+              uses: actions/checkout@v2
+              with:
+                  submodules: true
+                  fetch-depth: 0
+            - name: Check latest v8 stable version
+              run: cd deps && ./check_latest_v8_version.py
```

Comment:
I'm not sure if using Python for checking the different version is the most optimal. One thing that I wasn't is how to install the `requests` library inside the GitHub action.

Maybe using something like this:

```
- name: Set up Python 3.8
      uses: actions/setup-python@v2
      with:
        python-version: 3.8
    - name: Install dependencies
      run: |
        python -m pip install --upgrade pip
        pip install requests
```

But I'm not sure if it is a little overkill having to pull some python library to do the version check.

I have use Python, because most scripts in the repo are written in Python.

Open to suggestions 😄

Reply:
I know the requests library is popular, but the python standard library has http support.  Looks like the urllib.request and json libraries can be used for this.  For instance, the following worked for me

```python
import urllib.request
import json

CHROME_VERSIONS_URL = 'https://omahaproxy.appspot.com/all.json?os=linux&channel=stable'
with urllib.request.urlopen(CHROME_VERSIONS_URL) as response:
   json_response = response.read()

data = json.loads(json_response)
stable_version = data[0]["versions"][0]["v8_version"]
print(stable_version)
```